src/js/views/globals/search/patient-search.scss

```diff
@@ -35,6 +60,20 @@
   border-bottom: 1px solid $black-90;
 }
 
+.patient-search__search-by-button {
+  background-color: color(light_blue, light);
+  border: none;
+  color: $black-20;
+  font-size: 14px;
+  height: 28px;
```

Comment:
maybe we can reach a 4px increment, or avoiding setting the height at all by changing the line-height?